gateway-service/src/test/java/org/zowe/apiml/acceptance/PassTicketSchemeTest.java

```diff
@@ -56,7 +56,7 @@ void thenPassTicketErrorMessageIsReturned() {
                     .get(basePath + serviceWithDefaultConfiguration.getPath())
                     .then()
                     .log().all()
-                    .statusCode(is(HttpStatus.SC_INTERNAL_SERVER_ERROR)).body("messages[0].messageKey",is("org.zowe.apiml.security.ticket.generateFailed"));
+                    .statusCode(is(HttpStatus.SC_INTERNAL_SERVER_ERROR)).body("messages[0].messageKey",is("org.zowe.apiml.common.internalRequestError"));
```

Comment:
Why don't you provide more details to the consumer of the API?

Reply:
You are right this is not correct.